gulpfile.js

```diff
@@ -247,28 +195,42 @@ gulp.task('set', function() {
 
 });
 
-gulp.task('start-neo4j', ['set-password'], function() {
-  if(isWin) {
-    return runPowershell(neo4jHome + '/bin/neo4j.bat install-service;' + neo4jHome + '/bin/neo4j.bat start');
-  }
-  else {
-    return gulp.src('').pipe(shell([
-      'mkdir -p ' + neo4jHome + '/logs',
-      'mkdir -p ' + neo4jHome + '/run',
-      'mkdir -p ' + neo4jHome + '/plugins',
-      'mkdir -p ' + neo4jHome + '/import',
-      'chmod +x ' + neo4jHome + '/bin/neo4j',
-      neo4jHome + '/bin/neo4j start',
-    ]));
-  }
+
+var neo4jHome  = path.resolve('./build/neo4j');
+var neorunPath = path.resolve('./neokit/neorun.py');
+
+gulp.task('start-neo4j', function() {
+    var neorunArgs;
+    //var neorunArgs = minimist(process.argv.slice(2), { string: 'opts' }).opts;
+    if (neorunArgs == null)
```

Comment:
How can `neoRunArgs` not be `null` here? Typically the more ideomatic way is doing

```
neoRunArgs = neoRunArgs || '-v 3.0.1 -p neo4j'
```